packages/theme/src/theme/_modals.scss

```diff
@@ -1,36 +1,41 @@
-
 .modal-dialog {
 	max-height: 80vh;
 	display: flex;
 	flex-direction: column;
 }
 
 .modal {
-
+	&-content {
```

Comment:
May it be placed after `&-header`? (nice to have)

Reply:
no, because header is inside content